src/components/Search/CompareWithBase.tsx

```diff
@@ -201,6 +206,10 @@ function CompareWithBase({
 
   return (
     <Grid className={`wrapper--withbase ${wrapperStyles.wrapper}`}>
+      <SearchViewInit
+        repositoryBase={baseRepository}
+        repositoryNew={newRepository}
+      />
```

Comment:
I moved it here because this needs the repository information. But in a future patch this component will be removed.